backend/services/interfaces/testService.ts

```diff
@@ -0,0 +1,31 @@
+import { Question } from "../../models/test.model";
+import { UserDTO } from "../../types";
+
+export type TestResponseDTO = {
+  /** the unique identifier of the response */
+  id: string;
+  /** the name of the student */
+  name: string;
```

Comment:
`name` should refer to the name of the test (not an individual student)! please update the documentation to reflect this.